Controllers/controller.py

```diff
@@ -77,7 +80,35 @@ def open_file_dialog(self):
             mime_types.append(avi_video_file)
         elif mp4_video_file not in mime_types:
             mime_types.append(mp4_video_file)
-        file_dialog.setMimeTypeFilters(mime_types)
+
+        # Uses list of mime types to get filenames.
+        for i, item in enumerate(mime_types):
+            temp = ""
+            temp = item.removeprefix("audio/")
+            if temp == item:
+                temp = item.removeprefix("video/")
+            mime_types[i] = temp
+
+        # Appends the correct file types to a new list based on the Mime Types list.
+        filtered_types = []
+        for i in mime_types:
+            if i == "quicktime":
+                filtered_types.append("QuickTime video (*.mov)")
+            elif i == "mp4":
+                filtered_types.append("MP4 video (*.mp4)")
+            elif i == "x-msvideo":
+                filtered_types.append("AVI video (*.avi)")
+            else:
+                continue
+
+        # Appends .wmv and a default option that allows all files to be selected.
+        filtered_types.append("WMV video (*.wmv)")
```

Comment:
The default mime types should include every currently supported mime type and should not default to these static values.